docs/tutorials/istio.md

```diff
@@ -0,0 +1,28 @@
+# Configure Istio Service Mesh
```

Comment:
```suggestion
# Integrate Coder with Istio
```

Next will use whatever manifest.json tells it to, so the H1 here feels like it's more of just a reference for contributors